01.fizzbuzz/FizzBuzz.rb

```diff
@@ -0,0 +1,14 @@
+# 1から20までの数をプリントするプログラムを書け。ただし3の倍数のときは数の代わりに｢Fizz｣と、5の倍数のときは｢Buzz｣とプリントし、3と5両方の倍数の場合には｢FizzBuzz｣とプリントすること。
+
+1.upto(20) {|num|
```

Comment:
```suggestion
1.upto(20) { |num|
```

のようにスペースを空けることが多いです。
あと、Rubyの慣習として、複数行のブロックは`{}`よりも`do end`を使うことの方が多いです。

Reply:
ありがとうございます。
Rubyの慣習に習って、`do end`を使用しました。